runtime/amplitude/src/lib.rs

```diff
@@ -609,10 +565,15 @@ impl pallet_session::Config for Runtime {
 	type WeightInfo = pallet_session::weights::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+	// as per documentation, typical value for this is false "unless this pallet is being augmented by another pallet"
```

Comment:
Can you add the documentation link, here?

Reply:
Of course!